trc/mod_tracers.F90

```diff
@@ -100,6 +100,7 @@ subroutine allocate_tracers
 
      ! Number of HAMOCC tracers.
 #ifdef HAMOCC
+     call init_indices()
```

Comment:
I would find it preferable if the `init_indices` routine would be called in `hamocc_init`, but I guess this isn't possible because this routine is called first? But we should at least have a comment in `hamocc_init` saying that the initialization of HAMOCC indices is done here.

Reply:
Good point. I'll add that statement there.